kuyruk_sentry.py

```diff
@@ -13,7 +13,7 @@
 class Sentry(object):
     def __init__(self, kuyruk):
         kuyruk.extensions["sentry"] = self
-        self.client = raven.Client(kuyruk.config.SENTRY_DSN)
+        sentry_sdk.init(kuyruk.config.SENTRY_DSN)
```

Comment:
This global intialization may interfere with the other applications in the same process.